src/api/server/server.js

```diff
@@ -37,19 +69,19 @@ function getLedgerVersion(): number {
   return this.remote.getLedgerSequence();
 }
 
-function connect() {
+function connect(): Promise {
   return common.promisify(callback => {
     this.remote.connect(() => callback(null));
   })();
 }
 
-function disconnect() {
+function disconnect(): Promise {
   return common.promisify(callback => {
     this.remote.disconnect(() => callback(null));
   })();
 }
 
-function getServerInfo() {
+function getServerInfo(): Promise {
```

Comment:
Is there any way to annotate what the promise returns? I didn't see much in the docs about it.